### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RikuVirtanen**
### Changed
- Added StyleCop.Analyzers to analyze the Task code.
- Made code changes suggested by the StyleCop.Analyzers.
- Fixed Readme.

### CODE REVIEW

**Comment by RikuVirtanen** [comment]
Detect-Errors fails because of too broad Catch elements, but they are needed for the Task to work correctly.

**Comment by RikuVirtanen** [comment]
Detect-Errors fails because of too broad Catch elements, but they are needed for the Task to work correctly.

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ErrorTests.cs [hunk 1]
```diff
@@ -172,7 +162,8 @@ public void DownloadFiles_TestCancellationToken()
                 Operation = SourceOperation.Nothing,
             };
 
-            var ex = Assert.ThrowsAsync<Exception>(async () => await SFTP.DownloadFiles(source, _destination, connection, _options, _info, new CancellationTokenSource(2000).Token));
+            using var cancellationTokenSource = new CancellationTokenSource(2000);
+            var ex = Assert.ThrowsAsync<Exception>(async () => await SFTP.DownloadFiles(source, _destination, connection, _options, _info, cancellationTokenSource.Token));
             Assert.IsTrue(ex.Message.Contains("No files transferred."));
             Assert.IsTrue(ex.Message.Contains("Error: The operation was canceled.."));
             Assert.IsTrue(Helpers.SourceFileExists($"{source.Directory}/{source.FileName}"));
```
Comment (Svenskapojkarna) [inline Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ErrorTests.cs:169]: https://github.com/FrendsPlatform/Frends.SFTP/blob/ff45de45718f2d57ed3ea332c72aa78c993eed9f/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ErrorTests.cs#L197
No need to write to console.

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs [hunk 2]
```diff
@@ -1,9 +1,9 @@
-﻿using System.Text;
+﻿namespace Frends.SFTP.DownloadFiles.Definitions;
+
+using System.Text;
```
Comment (Svenskapojkarna) [inline Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs:3]: This namespace is not used, so it can be removed.

**Review by Svenskapojkarna — APPROVED** [review]

**Pull request merged**